common/app/views/fragments/commercial/adSlot.scala.html

```diff
@@ -6,18 +6,24 @@
     refresh: Boolean = true,
     outOfPage: Boolean = false,
     optId: Option[String] = None,
-    optClassNames: Option[String] = None
+    optClassNames: Option[String] = None,
+    adSlotToBlockthroughUids: Map[String, String] = Map("right" -> "5a9858a84f-157", "top-above-nav" -> "5a98585772-157")
```

Comment:
I wonder if these could be properties of slots defined in the Commercial helper object instead? https://github.com/guardian/frontend/blob/master/common/app/views/support/Commercial.scala